synapse/handlers/room_list.py

```diff
@@ -87,7 +93,8 @@ def get_local_public_room_list(self, limit=None, since_token=None,
     @defer.inlineCallbacks
     def _get_public_room_list(self, limit=None, since_token=None,
                               search_filter=None,
-                              network_tuple=EMPTY_THIRD_PARTY_ID,):
+                              network_tuple=EMPTY_THIRD_PARTY_ID,
+                              timeout=0,):
```

Comment:
`None` is probably better as a magic value than 0, fwiw

Reply:
Probs